### ISSUE TITLE
adding parameter estimates + CIs

### ISSUE THREAD

**Issue by IndrajeetPatil** [IndrajeetPatil/tidyBF#3]
```r
# setup
set.seed(123)
library(parameters)
library(BayesFactor)
#> Loading required package: coda
#> Loading required package: Matrix
#> ************
#> Welcome to BayesFactor 0.9.12-4.2. If you have questions, please contact Richard Morey (richarddmorey@gmail.com).
#> 
#> Type BFManual() to open the manual.
#> ************
library(bayestestR)

# one sample t test
mod1 <- ttestBF(x = rnorm(100, 1, 1))

model_parameters(mod1)
#> Parameter  | Median |       89% CI |   pd | % in ROPE |              Prior | Effects |   Component |    BF
#> ----------------------------------------------------------------------------------------------------------
#> Difference |   0.85 | [0.66, 1.05] | 100% |        0% | Cauchy (0 +- 0.71) |   fixed | conditional | > 999

#describe_posterior(mod1)

# independent t test
mod2 <- ttestBF(formula = wt ~ am, data = mtcars)

model_parameters(mod2)
#> Parameter  | Median |       89% CI |   pd | % in ROPE |              Prior | Effects |   Component |    BF
#> ----------------------------------------------------------------------------------------------------------
#> Difference |   1.26 | [0.80, 1.69] | 100% |        0% | Cauchy (0 +- 0.71) |   fixed | conditional | > 999

#describe_posterior(mod2)

# paired t test
mod3 <- ttestBF(
  x = sleep$extra[sleep$group == 1],
  y = sleep$extra[sleep$group == 2],
  paired = TRUE
)

model_parameters(mod3)
#> Parameter  | Median |       89% CI |   pd | % in ROPE |              Prior | Effects |   Component |    BF
#> ----------------------------------------------------------------------------------------------------------
#> Difference |   1.72 | [0.62, 3.31] | 100% |        0% | Cauchy (0 +- 0.71) |   fixed | conditional | 17.26

#describe_posterior(mod3)

mod4 <- correlationBF(y = iris$Sepal.Length, x = iris$Sepal.Width)

model_parameters(mod4)
#> Parameter | Median |        89% CI |     pd | % in ROPE |              Prior | Effects |   Component |   BF
#> -----------------------------------------------------------------------------------------------------------
#> rho       |  -0.11 | [-0.24, 0.00] | 92.12% |    43.27% | Cauchy (0 +- 0.33) |   fixed | conditional | 0.51

#describe_posterior(mod4)

data("raceDolls")
mod5 <-
  contingencyTableBF(
    raceDolls,
    sampleType = "indepMulti",
    fixedMargin = "cols"
  )

model_parameters(mod5)
#> Error in UseMethod("point_estimate"): no applicable method for 'point_estimate' applied to an object of class "NULL"

#describe_posterior(mod5)

mod6 <-
  anovaBF(
    formula = extra ~ group + ID,
    data = sleep,
    whichRandom = "ID",
    progress = FALSE
  )

model_parameters(mod6)
#> # Extra Parameters 
#> 
#> Parameter | Median |       89% CI |     pd | % in ROPE |    BF
#> --------------------------------------------------------------
#> mu        |   1.55 | [0.57, 2.47] | 98.88% |        0% | 11.59
#> sig2      |   1.02 | [0.33, 2.06] |   100% |        0% | 11.59
#> g_group   |   0.86 | [0.02, 5.99] |   100% |     3.85% | 11.59
#> g_ID      |   2.55 | [0.09, 6.65] |   100% |        0% | 11.59
#> 
#> # Fixed Effects 
#> 
#> Parameter | Median |         89% CI |     pd | % in ROPE |    BF
#> ----------------------------------------------------------------
#> group-1   |  -0.68 | [-1.07, -0.27] | 99.58% |        0% | 11.59
#> group-2   |   0.68 | [ 0.27,  1.07] | 99.58% |        0% | 11.59
#> 
#> # Random Effects 
#> 
#> Parameter | Median |         89% CI |     pd | % in ROPE |    BF
#> ----------------------------------------------------------------
#> ID-1      |  -0.19 | [-1.61,  1.06] | 59.33% |    10.87% | 11.59
#> ID-2      |  -1.55 | [-2.89, -0.13] | 96.62% |        0% | 11.59
#> ID-3      |  -0.88 | [-2.32,  0.40] | 84.97% |     5.81% | 11.59
#> ID-4      |  -1.66 | [-3.07, -0.29] | 97.20% |        0% | 11.59
#> ID-5      |  -1.33 | [-2.69,  0.08] | 93.17% |     1.49% | 11.59
#> ID-6      |   1.88 | [ 0.23,  3.19] | 98.28% |        0% | 11.59
#> ID-7      |   2.46 | [ 0.92,  3.91] | 99.58% |        0% | 11.59
#> ID-8      |  -0.28 | [-1.54,  1.13] | 63.18% |    11.15% | 11.59
#> ID-9      |   0.59 | [-0.78,  1.88] | 77.98% |     7.69% | 11.59
#> ID-10     |   0.92 | [-0.48,  2.19] | 87.58% |     4.97% | 11.59

#describe_posterior(mod6)

data("puzzles")
mod7 <-
  anovaBF(
    formula = RT ~ shape * color + ID,
    data = puzzles,
    whichRandom = "ID",
    whichModels = "top",
    progress = FALSE
  )

model_parameters(mod7)
#> # Extra Parameters 
#> 
#> Parameter | Median |         89% CI |   pd | % in ROPE |   BF
#> -------------------------------------------------------------
#> mu        |  44.99 | [43.92, 46.02] | 100% |        0% | 2.76
#> sig2      |   1.77 | [ 1.14,  2.51] | 100% |        0% | 0.25
#> g_shape   |   0.35 | [ 0.02,  2.29] | 100% |    13.17% | 2.76
#> g_color   |   0.32 | [ 0.02,  2.07] | 100% |    13.65% | 0.25
#> g_ID      |   2.61 | [ 0.78,  5.06] | 100% |        0% | 0.25
#> 
#> # Fixed Effects 
#> 
#> Parameter           | Median |         89% CI |     pd | % in ROPE |   BF
#> -------------------------------------------------------------------------
#> shape-round         |   0.42 | [ 0.12,  0.74] | 98.83% |        0% | 0.25
#> shape-square        |  -0.42 | [-0.74, -0.12] | 98.83% |        0% | 0.25
#> color-color         |  -0.43 | [-0.72, -0.12] | 98.98% |        0% | 2.76
#> color-monochromatic |   0.43 | [ 0.12,  0.72] | 98.98% |        0% | 0.25
#> 
#> # Random Effects 
#> 
#> Parameter | Median |         89% CI |     pd | % in ROPE |   BF
#> ---------------------------------------------------------------
#> ID-1      |   2.46 | [ 1.03,  3.85] | 99.72% |        0% | 0.25
#> ID-2      |   0.43 | [-0.92,  1.91] | 70.43% |     8.79% | 2.76
#> ID-3      |   0.91 | [-0.55,  2.34] | 84.72% |     5.78% | 0.25
#> ID-4      |   0.47 | [-1.07,  1.83] | 69.83% |     9.32% | 0.25
#> ID-5      |   3.14 | [ 1.68,  4.52] | 99.95% |        0% | 2.76
#> ID-6      |   0.45 | [-0.93,  1.94] | 70.08% |     9.38% | 0.25
#> ID-7      |  -3.14 | [-4.58, -1.73] | 99.92% |        0% | 0.25
#> ID-8      |  -0.23 | [-1.58,  1.18] | 60.77% |     9.04% | 2.76
#> ID-9      |  -2.47 | [-3.97, -1.10] | 99.62% |        0% | 0.25
#> ID-10     |   0.67 | [-0.70,  2.12] | 77.95% |     6.74% | 0.25
#> ID-11     |   0.66 | [-0.75,  2.05] | 78.40% |     7.10% | 2.76
#> ID-12     |  -3.38 | [-4.77, -1.89] | 99.98% |        0% | 0.25

#describe_posterior(mod7)

mod8 <- regressionBF(rating ~ ., data = attitude, progress = FALSE)

model_parameters(mod8)
#> # Extra Parameters component
#> 
#> Parameter | Median |         89% CI |   pd | % in ROPE | Effects
#> ----------------------------------------------------------------
#> mu        |  64.63 | [62.54, 66.73] | 100% |        0% |   fixed
#> sig2      |  52.12 | [30.84, 76.69] | 100% |        0% |   fixed
#> g         |   1.29 | [ 0.05,  8.83] | 100% |     0.31% |   fixed
#> 
#> # Fixed Effects component
#> 
#> Parameter  | Median |       89% CI |   pd | % in ROPE | Effects
#> ---------------------------------------------------------------
#> complaints |   0.73 | [0.56, 0.90] | 100% |        0% |   fixed

#describe_posterior(mod8)

## Bem's (2010) data (see Rouder & Morey, 2011)
t <- c(-.15, 2.39, 2.42, 2.43)
N <- c(100, 150, 97, 99)

## Using rscale=1 and one-sided test to be
## consistent with Rouder & Morey (2011)
mod9 <- meta.ttestBF(t, N, rscale = 1, nullInterval = c(0, Inf))

model_parameters(mod9)
#> Parameter | Median |       89% CI |   pd | % in ROPE |           Prior | Effects |   Component
#> ----------------------------------------------------------------------------------------------
#> Effect    |   0.17 | [0.10, 0.25] | 100% |     0.90% | Cauchy (0 +- 1) |   fixed | conditional

#describe_posterior(mod9)

data("puzzles")
mod10 <-
  lmBF(
    formula = RT ~ shape + color + shape:color + ID,
    data = puzzles,
    whichRandom = "ID",
    posterior = TRUE,
    iterations = 1000
  )

model_parameters(mod10)
#> Parameter                          | Median |         89% CI |     pd | % in ROPE
#> ---------------------------------------------------------------------------------
#> mu                                 |  45.02 | [44.01, 46.06] |   100% |        0%
#> shape-round                        |   0.42 | [ 0.12,  0.72] | 98.70% |     4.00%
#> shape-square                       |  -0.42 | [-0.72, -0.12] | 98.70% |     4.00%
#> color-color                        |  -0.44 | [-0.73, -0.13] | 98.60% |     3.80%
#> color-monochromatic                |   0.44 | [ 0.13,  0.73] | 98.60% |     3.80%
#> ID-1                               |   2.50 | [ 1.02,  3.89] | 99.70% |     0.10%
#> ID-2                               |   0.45 | [-0.88,  1.83] | 70.30% |     8.60%
#> ID-3                               |   0.94 | [-0.46,  2.40] | 85.90% |     4.70%
#> ID-4                               |   0.48 | [-1.09,  1.79] | 71.80% |     6.50%
#> ID-5                               |   3.18 | [ 1.64,  4.53] |   100% |        0%
#> ID-6                               |   0.47 | [-0.88,  1.87] | 68.80% |     8.50%
#> ID-7                               |  -3.15 | [-4.65, -1.79] | 99.90% |     0.10%
#> ID-8                               |  -0.23 | [-1.66,  1.11] | 60.00% |     9.50%
#> ID-9                               |  -2.41 | [-3.92, -1.03] | 99.40% |     0.40%
#> ID-10                              |   0.68 | [-0.71,  2.03] | 79.20% |     7.40%
#> ID-11                              |   0.62 | [-0.82,  1.94] | 77.70% |     6.90%
#> ID-12                              |  -3.37 | [-4.81, -1.82] |   100% |        0%
#> shape:color-round.&.color          |  -0.01 | [-0.29,  0.22] | 53.80% |    49.20%
#> shape:color-round.&.monochromatic  |   0.01 | [-0.22,  0.29] | 53.80% |    49.20%
#> shape:color-square.&.color         |   0.01 | [-0.22,  0.29] | 53.80% |    49.20%
#> shape:color-square.&.monochromatic |  -0.01 | [-0.29,  0.22] | 53.80% |    49.20%
#> sig2                               |   1.75 | [ 1.18,  2.55] |   100% |        0%
#> g_shape                            |   0.36 | [ 0.02,  2.20] |   100% |    11.20%
#> g_color                            |   0.34 | [ 0.02,  1.95] |   100% |    11.70%
#> g_ID                               |   2.56 | [ 0.84,  5.14] |   100% |        0%
#> g_shape:color                      |   0.22 | [ 0.02,  1.40] |   100% |    23.50%

#describe_posterior(mod10)
```

**Comment by IndrajeetPatil** [comment]
New `bf_extractor`

```r
options(tibble.width = Inf)
library(magrittr)

broomExtra::tidy_parameters(bf.object) %>%
  dplyr::rename(.data = ., bf10 = bf) %>%
  tidyBF:::bf_formatter(.)
```

**Comment by IndrajeetPatil** [comment]
Closed in favor of #6
